tools/misc/tpm2_checkquote.c

```diff
@@ -57,6 +57,7 @@ static tpm2_verifysig_ctx ctx = {
 static bool verify(void) {
 
     bool result = false;
+    EVP_PKEY_CTX *pkey_ctx = NULL;
```

Comment:
Define this at line 78, closest to its first use.

Reply:
@idesai  EVP_PKEY_set_alias_type(pkey, EVP_PKEY_SM2)  must be called before EVP_PKEY_CTX_new,  if EVP_PKEY_set_alias_type fail, goto err and free the pkey_ctx, so initialize pkey_ctx to NULL.